### PR TITLE
Go to user profile in channel list.

### PR DESCRIPTION
**Pull request by lyw0149**
1. Go to user profile directly in channel list.
2. Same profile in same chatPage.
3. Same loading animation.

### CODE REVIEW

**Comment by ifndefdeadmau5** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/146/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

File: src/components/Chat/react-native-gifted-chat/src/LoadEarlier.js [hunk 1]
```diff
@@ -19,18 +19,10 @@ export default class LoadEarlier extends React.Component {
...
-        <ActivityIndicator
-          color='white'
-          size='small'
-          style={[styles.activityIndicator, this.props.activityIndicatorStyle]}
-        />
-      </View>
+        <View style={styles.loadingContainer}>
+          <ActivityIndicator animating={!this.state.loaded} size="large" />
+          <Text>Loading...</Text>
+        </View>
```
Comment (ifndefdeadmau5) [inline src/components/Chat/react-native-gifted-chat/src/LoadEarlier.js]: Indent got deeper?

Comment (lyw0149) [reply src/components/Chat/react-native-gifted-chat/src/LoadEarlier.js]: OMG eagle eye.

File: src/components/Chat/ChannelList.js [hunk 2]
```diff
@@ -139,15 +139,9 @@ class ChannelList extends Component {
...
-        <View style={styles.loadingViewheader}>
-          <Text style={styles.loadingViewheaderText}>Loading...</Text>
-          <ActivityIndicator
-            animating={true}
-            style={[styles.loadingViewActivityIndicator]}
-            size='large'
-          />
-        </View>
+      <View style={styles.loadingContainer}>
+        <ActivityIndicator animating={!this.state.loaded} size="large" />
```
Comment (ace0625) [inline src/components/Chat/ChannelList.js]: Can you change the indicator size to small?

File: src/components/Chat/react-native-gifted-chat/src/Avatar.js [hunk 3]
```diff
@@ -19,6 +19,7 @@ export default class Avatar extends React.Component {
         avatarStyle={StyleSheet
           .flatten([styles[this.props.position].image, this.props.imageStyle[this.props.position]])}
         user={this.props.currentMessage.user}
+        opponentImage={this.props.opponentInfo.profile_picture_small}
```
Comment (SarahhhC) [inline src/components/Chat/react-native-gifted-chat/src/Avatar.js:22]: We should check if there's no small picture in DB, if not use profile_picture instead.

File: src/components/Chat/react-native-gifted-chat/src/LoadEarlier.js [hunk 4]
```diff
@@ -19,17 +19,9 @@ export default class LoadEarlier extends React.Component {
...
-        }, ]}>
-          {this.props.label}
-        </Text>
-        <ActivityIndicator
-          color='white'
-          size='small'
-          style={[styles.activityIndicator, this.props.activityIndicatorStyle]}
-        />
+      <View style={styles.loadingContainer}>
+        <ActivityIndicator animating={!this.state.loaded} size="large" />
```
Comment (ace0625) [inline src/components/Chat/react-native-gifted-chat/src/LoadEarlier.js]: This one as well.

File: src/components/Chat/react-native-gifted-chat/src/Avatar.js (hunk 3 above)
Comment (lyw0149) [reply src/components/Chat/react-native-gifted-chat/src/Avatar.js:22]: No small_picture in DB is possible??

**Pull request merged**